src/h2load.cc

```diff
@@ -271,9 +345,32 @@ void Client::disconnect() {
 }
 
 void Client::submit_request() {
+  restart_timeout();
   auto req_stat = &worker->stats.req_stats[worker->stats.req_started++];
   session->submit_request(req_stat);
   ++req_started;
+
+  // if an active timeout is set and this is the last request to be submitted
+  // on this connection, start the active timeout.
+  if (worker->config->conn_active_timeout > 0 && req_started >= req_todo) {
+    ev_timer_start(worker->loop, &conn_active_watcher);
+  }
+}
+
+void Client::process_timedout_streams() {
+  for (auto req_stat : worker->stats.req_stats) {
```

Comment:
`&` is required for `req_stat`, otherwise it is a copy of each element in worker->stats.req_stats.
So, do like so:
```
for (auto &req_stat : worker->stats.req_stats) {
```